### PR TITLE
Add new strings and translate to PT-BR

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

File: src/locales/pt_br/translation.json [hunk 1]
```diff
@@ -977,6 +1043,26 @@
...
+    "amount": "Quantidade:",
+    "liquidationPrice": "Preço de liquidação:"
+  },
+  "openPositionTable": {
+    "direction": "Direção",
+    "positionSize": "Tamanho da posição",
+    "entryPrice": "Preço de entrada",
+    "liquidationPrice": "Preço de liquidação",
+    "positionMargin": "Posição margem",
+    "unrealizedPL": "P&L não realizado",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:1057]:
```suggestion
    "unrealizedPL": "LP não realizado",
```

File: src/locales/pt_br/translation.json [hunk 2]
```diff
@@ -86,19 +86,26 @@
...
     "auth": "Por favor, conecte e autorize sua carteira.",
     "noClosedTrades": "Você não tem operações fechadas.",
     "table": {
+      "direction": "Direção",
       "positionSize": "Tamanho da posição",
       "leverage": "Alavancagem",
       "openPrice": "Preço de entrada",
       "closePrice": "Preço de fechamento",
-      "profit": "Lucro / Prejuízo",
+      "profit": "P&L realizado",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:106]:
```suggestion
      "profit": "L&P realizado",
```

File: src/locales/pt_br/translation.json [hunk 3]
```diff
@@ -863,6 +883,51 @@
...
+    },
+    "tradeForm": {
+      "buy_cta": "COMPRAR",
+      "sell_cta": "VENDER",
+      "amountReceived": "Receber",
+      "buy": "COMPRA",
+      "sell": "VENDA"
+    },
+    "priceHistory": {
+      "turnover": "Turnover",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:922]:
```suggestion
      "turnover": "Volume de negociações",
```

**Comment by bitcoinuser** [comment]
Hi @creed-victor, could you please merge this? thanks.

File: src/locales/pt_br/translation.json [hunk 4]
```diff
@@ -863,6 +886,51 @@
...
+    "tradingHistory": "Histórico"
+  },
+  "spotTradingPage": {
+    "meta": {
+      "title": "Operar Spot",
+      "description": "Operar Spot"
+    },
+    "tradeForm": {
+      "buy_cta": "COMPRAR",
+      "sell_cta": "VENDER",
```
Comment (bitcoinuser) [inline src/locales/pt_br/translation.json:919]:
```suggestion
      "sell_cta": "VENDA",
```

**Review by creed-victor — APPROVED** [review]

**Pull request merged**